conftest.py

```diff
@@ -53,7 +53,10 @@ def component_cache(jp_environ):
     component_cache.load()
 
     yield component_cache
-    component_cache.cache_manager.stop()
+    if component_cache.is_server_process:
```

Comment:
@kiersten-stokes I need to follow-up with you. I [temporarily?] had to add this to prevent test failures that surfaced because the `cache_manager` attribute isn't defined. Not sure why this fixture isn't working the way it did before I added the new test cases.